tests/dummy/app/templates/application.hbs

```diff
@@ -6,6 +6,7 @@
 
   <div class="panel">
     {{simple-chart height=250 width=250 name='donut' data=donutData}}
+    {{simple-chart height=250 width=250 name='pie' data=pieData}}
```

Comment:
Can you stick this in its own 'pannel' div.  I think we will eventually style them to look nice in a grid.